### PR TITLE
RDM-13210 upload manage translations

### PR DESCRIPTION
**Pull request by RebeccaBaker**
### JIRA link ###
https://tools.hmcts.net/jira/browse/RDM-13210

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: config/custom-environment-variables.yaml [hunk 1]
```diff
@@ -33,6 +33,7 @@ adminWeb:
   elastic_case_types_url: ADMINWEB_ELASTIC_CASE_TYPES_URL
   global_search_index_url: ADMINWEB_GLOBAL_SEARCH_INDEX_URL
   welsh_translation_get_dictionary_url: ADMINWEB_GET_DICTIONARY_URL
+  welsh_translation_upload_dictionary_file_path: ADMINWEB_UPLOAD_DICTIONARY_FILE_PATH
```
Comment (RebeccaBaker) [inline config/custom-environment-variables.yaml:36]: I don't think this is still needed

File: package.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
     "applicationinsights": "^1.0.8",
     "body-parser": "^1.19.0",
     "config": "^2.0.1",
+    "convert-csv-to-json": "^1.3.3",
```
Comment (RebeccaBaker) [inline package.json:36]: do we still use this?

File: yarn.lock [hunk 3]
```diff
@@ -2015,6 +2045,11 @@ continuation-local-storage@^3.2.1:
     async-listener "^0.6.0"
     emitter-listener "^1.1.1"
 
+convert-csv-to-json@^1.3.3:
+  version "1.3.3"
+  resolved "https://registry.yarnpkg.com/convert-csv-to-json/-/convert-csv-to-json-1.3.3.tgz#0351a75dac840e1b9c690d193a8c9f87ab2f5199"
+  integrity sha512-lIQrCmEw0GCywtVWeJ/AGv5hhSTPsMFdqOf9GcMtActqO2Gt8z1RZgfw9NBOll479cIPM6BqDOFC8R3W7bSwtA==
+
```
Comment (RebeccaBaker) [inline yarn.lock:2052]: think this can be deleted too

File: src/main/service/manage-welsh-dictionary-service.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  const translations = buildTranslationsJson(data);
+  const dictionary = "{\"translations\":{" + translations + "}}";
+
+  return request
+        .put(url)
+        .set(headers)
+        .set("enctype", "multipart/form-data")
+        .send(dictionary)
+        .then((res) => {
+            logger.info(`Received successful response: ${res.text}`);         return res;
```
Comment (RebeccaBaker) [inline src/main/service/manage-welsh-dictionary-service.ts:55]: could be formatting in github ui but can we separate to two lines please

**Review by RebeccaBaker — COMMENTED** [review]
minor tidy up comments

File: config/custom-environment-variables.yaml (hunk 1 above)
Comment (markdathornehmcts) [reply config/custom-environment-variables.yaml:36]: Indeed

File: yarn.lock (hunk 3 above)
Comment (markdathornehmcts) [reply yarn.lock:2052]: Yes

File: src/main/service/manage-welsh-dictionary-service.ts (hunk 4 above)
Comment (markdathornehmcts) [reply src/main/service/manage-welsh-dictionary-service.ts:55]: Done

**Pull request closed without merging**